src/engine/sched.c

```diff
@@ -1624,6 +1634,9 @@ sched_watchdog_post(struct dss_xstream *dx, struct sched_unit *su)
 	unsigned int		 elapsed;
 	char			**strings;
 
+	/* A ULT is just scheduled, increase schedule seq */
+	info->si_cur_seq++;
```

Comment:
just confirm that if ULT_1 execute on XS1, it create another ULT_2 on XS2 and yield.
The ULT_2 execute on XS2 will cause XS2's dx->dx_sched_info-> si_cur_seq increased.
But on XS1,  if no other ULT/req scheduled on XS1 and original ULT_1 resume executing after yield, seems then its own si_cur_seq will not be changed. But in this case, seems we are fine (need not re-probe)?

Reply:
As long ULT_1 yield, and scheduler get control, the schedule sequence will be bumped, no matter if the ULT_1 is immediately scheduled again, right?

Reply:
Ah, ok, thanks.